tests/integ/scala/test_datatype_suite.py

```diff
@@ -341,7 +350,11 @@ def test_verify_datatypes_reference_vector(session):
     "config.getoption('local_testing_mode', default=False)",
     reason="FEAT: function to_geography not supported",
 )
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC, reason="Alter Session not supported in stored procedure."
+)
 def test_dtypes(session):
+    session.sql("alter session set feature_interval_types=enabled;").collect()
```

Comment:
I guess this can be enabled at session level instead of enabling and then disabling for each test.

https://github.com/snowflakedb/snowpark-python/blob/784c3d5c449d7ee94c0447a652e29bab69d80760/tests/integ/conftest.py#L272-L276

```
session = (
    Session.builder.configs(db_parameters)
    .config("local_testing", local_testing_mode)
    .config("session_parameters": {"feature_interval_types": "ENABLED"})
    .create()
)
```